<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managereaders</class>
 <widget class="QDialog" name="managereaders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>读者信息：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存修改</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>841</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>5</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>检索</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>按学工号查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按姓名查询</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除所选</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>811</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>15</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>学工号</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>读者姓名</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>院系</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类别</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
